cdap-ui/cypress/integration/pipeline.runtimeargs.spec.ts

```diff
@@ -289,6 +289,17 @@ describe('Deploying pipeline with temporary runtime arguments', () => {
       )}`
     ).should('not.exist');
   });
+
+  it('should fail running with invalid runtime arguments', () => {
```

Comment:
Should the test name specifically reference `profile`? Is profile handled differently from other runtime args?

Reply:
The profile can be a runtime argument. The reference to profile as runtime argument is just an example in the JIRA. It doesn't have to be profile.